documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -107,9 +121,98 @@ EOF
 <2> Specify the _managed object reference_ of the VMware host.
 <3> Specify the IP address of the VMware migration network.
 
+. If you want to map multiple networks, create a `NetworkAttachmentDefinition` CR manifest for each additional destination network:
```

Comment:
As this is not the native place to document NAD, I suggest to display this yaml as an example to NAD creation.

Reply:
I think I will turn this into a link to the OCP example and take out the yaml. Makes sense if we move this to the prerequisites.